workers/unity/Assets/Fps/Scripts/GameLogic/WorldTiles/TileCollapser.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections.Generic;
+using MeshUtilities;
+using UnityEngine;
+
+public class TileCollapser : MonoBehaviour
+{
+    private readonly Dictionary<string, CombinedMeshAndMaterialsData> collapsedInstances =
+        new Dictionary<string, CombinedMeshAndMaterialsData>();
+
+    private void Awake()
+    {
+        for (var i = 0; i < transform.childCount; i++)
+        {
+            Debug.Log(transform.childCount);
```

Comment:
This log may not be necessary (the other one in this file makes sense but may also be unnecessary)